docs/usage/configuration-options.md

```diff
@@ -205,7 +209,9 @@ In that case Renovate first creates a branch and associated Pull Request, and th
 If by the next run the PR is already behind the base branch it will be automatically rebased, because Renovate only automerges branches which are up-to-date and green.
 If Renovate is scheduled for hourly runs on the repository but commits are made every 15 minutes to the main branch, then an automerge like this will keep getting deferred with every rebase.
 
-Note: if you have no tests but still want Renovate to automerge, you need to add `"ignoreTests": true` to your configuration.
+<!-- prettier-ignore -->
+!!! note
+    If you have no tests but still want Renovate to automerge, you need to add `"ignoreTests": true` to your configuration.
```

Comment:
```suggestion
<!-- prettier-ignore -->
!!! tip
    If you have no tests but still want Renovate to automerge, you need to add `"ignoreTests": true` to your configuration.
```

This might be better as a `tip` style admonition.